sops/3scale-scaling.md

```diff
@@ -0,0 +1,65 @@
+# 3scale scaling on Openshift 3.11, Integreatly 1.4.x
+This guide states the known limits of 3scale capacity on Openshift along with scaling options for 3scale Components.
```

Comment:
The main audience for this guide will be Cloud Services Eng and SRE teams. Perhaps opening with something like "The purpose of this guide is to outline the existing scaling capabilities of 3Scale, including scaling options for high availability and performance driven implementations along with some known limitations within the product."